.github/workflows/build-push.yml

```diff
@@ -0,0 +1,28 @@
+name: Tutor Build Action Test
+
+on:
+  merge:
+    branches:
+      - 'pearson-release/olive.main'
+      - 'pearson-release/olive.stage'
+
+jobs:
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+
+      - name: Build Image
+        uses: Pearson-Advance/tutor-build-image-action@vue/PADV-325
+        with:
+          python-version: '3.8'
+          tutor-version: '15'
+          path-image-to-be-built: 'env/plugins/ecommerce/build/ecommerce'
+          repository-name: 'pearsonopenedxops'
+          image-name: 'tutor-ecommerce-test'
+          image-tag: '0.0.1'
```

Comment:
I think it has to be changed in every new PR candidate to be merged, I don't know if it can be changed automatically when is merged. I'm going to investigate that.

Reply:
so the image tag is automatically updated?

Reply:
I think is better to specify the image tag in a repo variable, so it can be updated easily